File: simple-stack-example-mvp-view/src/main/java/com/zhuinden/simplestackdemoexamplemvp/util/Strings.java
package com.zhuinden.simplestackdemoexamplemvp.util;

import android.support.annotation.Nullable;

/**
 * Created by Owner on 2017. 01. 27..
 */

public class Strings {
    public static boolean isNullOrEmpty(@Nullable String string) {
        return string == null || string.length() == 0;
    }
}

File: simple-stack-example-mvp-fragments/src/main/java/com/zhuinden/simplestackdemoexamplefragments/util/Strings.kt
package com.zhuinden.simplestackdemoexamplefragments.util

/**
 * Created by Zhuinden on 2018. 08. 20.
 */

object Strings {
    @JvmStatic
    fun isNullOrEmpty(string: String?): Boolean {
        return string == null || string.isEmpty()
    }
}
